packages/libs/react-ui/src/components/Form/TextField/TextField.tsx

```diff
@@ -1,29 +1,58 @@
 import type { IFormFieldWrapperProps, IInputProps } from '@components/Form';
-import { FormFieldWrapper, Input } from '@components/Form';
+import { Input } from '@components/Form';
+import { Stack } from '@components/Layout';
 import type { FC } from 'react';
 import React from 'react';
+import { FormFieldHeader } from '../FormFieldWrapper/FormFieldHeader/FormFieldHeader';
+import { FormFieldHelper } from '../FormFieldWrapper/FormFieldHelper/FormFieldHelper';
+import { statusVariant } from '../FormFieldWrapper/FormFieldWrapper.css';
 
 export interface ITextFieldProps
-  extends Omit<IFormFieldWrapperProps, 'children' | 'htmlFor'> {
-  inputProps: Omit<IInputProps, 'disabled' | 'children' | 'leadingTextWidth'>;
-}
-
+  extends Omit<IFormFieldWrapperProps, 'children' | 'htmlFor'>,
+    Omit<IInputProps, 'disabled' | 'children'> {}
+// FormField with action buttons? pass the children as its by default?
 export const TextField: FC<ITextFieldProps> = ({
   disabled = false,
-  inputProps,
   status,
+  leadingText,
+  startIcon,
+  leadingTextWidth,
+  type,
+  ref,
+  id,
```

Comment:
props that are intended for accessibility can be removed from props since react-aria should provide them